UPGRADE-7.0.md

```diff
@@ -199,20 +240,25 @@ FrameworkBundle
    $this->render(..., ['form' => $form]);
    ```
 
- * Remove the integration of Doctrine annotations, use native attributes instead
+ * Remove the integration of the Doctrine annotations library, use native attributes instead
  * Remove `EnableLoggerDebugModePass`, use argument `$debug` of HttpKernel's `Logger` instead
  * Remove `AddDebugLogProcessorPass::configureLogger()`, use HttpKernel's `DebugLoggerConfigurator` instead
- * Make the `framework.handle_all_throwables` config option default to `true`
- * Make the `framework.php_errors.log` config option default to `true`
- * Make the `framework.session.cookie_secure` config option default to `auto`
- * Make the `framework.session.cookie_samesite` config option default to `lax`
- * Make the `framework.session.handler_id` default to null if `save_path` is not set and to `session.handler.native_file` otherwise
- * Make the `framework.uid.default_uuid_version` config option default to `7`
- * Make the `framework.uid.time_based_uuid_version` config option default to `7`
- * Make the `framework.validation.email_validation_mode` config option default to `html5`
+ * Add `array $tokenAttributes = []` optional parameter to `KernelBrowser::loginUser()`
+ * Change default of some config options:
+
+   | option                                       | default Symfony <7.0     | default in Symfony 7.0+                                                   |
+   | -------------------------------------------- | ------------------------ | ------------------------------------------------------------------------- |
+   | `framework.http_method_override`             | `true`                   | `false`                                                                   |
+   | `framework.handle_all_throwables`            | `false`                  | `true`                                                                    |
+   | `framework.php_errors.log`                   | `kernel.debug`           | `true`                                                                    |
+   | `framework.session.cookie_secure`            | `false`                  | `auto`                                                                    |
+   | `framework.session.cookie_samesite`          | `lax`                    | `lax`                                                                     |
```

Comment:
I'm a bit unsure about this deprecation. As you can see, I believe we did not change the default value between 6.4 and 7.0. Either I'm missing something, or we incorrectly force everyone to set this config option on 6.4?

Reply:
```suggestion
   | `framework.session.cookie_samesite`          | `null`                    | `lax`                                                                     |
```
We did change it from `null` to `lax` in https://github.com/symfony/symfony/commit/4fe6f5b2bc41ad515998a4d977b3158ddbd6fe98#diff-041b166c698c254a3f61cfc6390785d4f5097794b77a4b2f3f888996729cbce4L669